lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -340,6 +340,35 @@ sub add_batch_job {
     return $batch_id;
 }
 
+sub build_process_unfinished_tests_select_query {
+    my ( $self ) = @_;
+    
+    if ($self->config->lock_on_queue()) {
+        return "
+            SELECT hash_id, results, nb_retries
+            FROM test_results 
+            WHERE test_start_time < DATE_SUB(NOW(), INTERVAL ".$self->config->timeout_before_retrying_test_in_minutes()." MINUTE) 
```

Comment:
Would it be possible to use `?` instead of string concatenation in cases like this? That way we'd be more resilient against SQL-injection attacks.

Reply:
I don't think it is possible in this specific case. But not 100% sure.

Reply:
Fair enough.